From 23e0b93c3bbe96a365357a3af11bc86172102c05 Mon Sep 17 00:00:00 2001
From: Barbora Simonova <bsmejkal@redhat.com>
Date: Wed, 25 Jun 2025 10:43:37 +0200
Subject: [PATCH] Issue 6829 - Update parametrized docstring for tests

Description:
Update missing parametrized value in docstring for some tests

Fixes: https://github.com/389ds/389-ds-base/issues/6829

Reviewed by: @vashirov (Thanks!)
---
 dirsrvtests/tests/suites/basic/basic_test.py              | 2 +-
 dirsrvtests/tests/suites/clu/dsconf_config_test.py        | 8 ++++++++
 dirsrvtests/tests/suites/clu/schema_test.py               | 5 +++++
 dirsrvtests/tests/suites/mapping_tree/regression_test.py  | 1 +
 dirsrvtests/tests/suites/password/password_test.py        | 1 +
 .../tests/suites/replication/regression_m2_test.py        | 1 +
 dirsrvtests/tests/suites/vlv/regression_test.py           | 2 ++
 7 files changed, 19 insertions(+), 1 deletion(-)

diff --git a/dirsrvtests/tests/suites/basic/basic_test.py b/dirsrvtests/tests/suites/basic/basic_test.py
index 8f5de91aa..db6bfab56 100644
--- a/dirsrvtests/tests/suites/basic/basic_test.py
+++ b/dirsrvtests/tests/suites/basic/basic_test.py
@@ -961,7 +961,7 @@ def test_basic_search_lookthroughlimit(topology_st, limit, resp, import_example_
     Tests normal search with lookthroughlimit set high and low.
 
     :id: b5119970-6c9f-41b7-9649-de9233226fec
-
+    :parametrized: yes
     :setup: Standalone instance, add example.ldif to the database, search filter (uid=*).
 
     :steps:
diff --git a/dirsrvtests/tests/suites/clu/dsconf_config_test.py b/dirsrvtests/tests/suites/clu/dsconf_config_test.py
index d67679adf..232018097 100644
--- a/dirsrvtests/tests/suites/clu/dsconf_config_test.py
+++ b/dirsrvtests/tests/suites/clu/dsconf_config_test.py
@@ -58,6 +58,7 @@ def test_single_value_add(topology_st, attr_name, values_dict):
     """Test adding a single value to an attribute
 
     :id: ffc912a6-c188-413d-9c35-7f4b3774d946
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add a single value to the specified attribute
@@ -89,6 +90,7 @@ def test_single_value_replace(topology_st, attr_name, values_dict):
     """Test replacing a single value in configuration attributes
 
     :id: 112e3e5e-8db8-4974-9ea4-ed789c2d02f2
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add initial value to the specified attribute
@@ -127,6 +129,7 @@ def test_multi_value_batch_add(topology_st, attr_name, values_dict):
     """Test adding multiple values in a single batch command
 
     :id: 4c34c7f8-16cc-4ab6-938a-967537be5470
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add multiple values to the attribute in a single command
@@ -157,6 +160,7 @@ def test_multi_value_batch_replace(topology_st, attr_name, values_dict):
     """Test replacing with multiple values in a single batch command
 
     :id: 05cf28b8-000e-4856-a10b-7e1df012737d
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add initial single value
@@ -194,6 +198,7 @@ def test_multi_value_specific_delete(topology_st, attr_name, values_dict):
     """Test deleting specific values from multi-valued attribute
 
     :id: bb325c9a-eae8-438a-b577-bd63540b91cb
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add multiple values to the attribute
@@ -232,6 +237,7 @@ def test_multi_value_batch_delete(topology_st, attr_name, values_dict):
     """Test deleting multiple values in a single batch command
 
     :id: 4b105824-b060-4f83-97d7-001a01dba1a5
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add multiple values to the attribute
@@ -269,6 +275,7 @@ def test_single_value_persists_after_restart(topology_st, attr_name, values_dict
     """Test single value persists after server restart
 
     :id: be1a7e3d-a9ca-48a1-a3bc-062990d4f3e9
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add single value to the attribute
@@ -310,6 +317,7 @@ def test_multi_value_batch_persists_after_restart(topology_st, attr_name, values
     """Test multiple values added in batch persist after server restart
 
     :id: fd0435e2-90b1-465a-8968-d3a375c8fb22
+    :parametrized: yes
     :setup: Standalone DS instance
     :steps:
         1. Add multiple values in a single batch command
diff --git a/dirsrvtests/tests/suites/clu/schema_test.py b/dirsrvtests/tests/suites/clu/schema_test.py
index 19ec032bc..5709471cf 100644
--- a/dirsrvtests/tests/suites/clu/schema_test.py
+++ b/dirsrvtests/tests/suites/clu/schema_test.py
@@ -100,6 +100,7 @@ def test_origins(create_attribute):
     """Test the various possibilities of x-origin
 
     :id: 3229f6f8-67c1-4558-9be5-71434283086a
+    :parametrized: yes
     :setup: Standalone Instance
     :steps:
         1. Add an attribute with different x-origin values/types
@@ -116,6 +117,7 @@ def test_mrs(create_attribute):
     """Test an attribute can be added with a matching rule
 
     :id: e4eb06e0-7f80-41fe-8868-08c2bafc7590
+    :parametrized: yes
     :setup: Standalone Instance
     :steps:
         1. Add an attribute with a matching rule
@@ -132,6 +134,7 @@ def test_edit_attributetype(create_attribute):
     """Test editing an attribute type in the schema
 
     :id: 07c98f6a-89f8-44e5-9cc1-353d1f7bccf4
+    :parametrized: yes
     :setup: Standalone Instance
     :steps:
         1. Add a new attribute type
@@ -209,6 +212,7 @@ def test_edit_attributetype_remove_superior(create_attribute):
     """Test editing an attribute type to remove a parameter from it
 
     :id: bd6ae89f-9617-4620-adc2-465884ca568b
+    :parametrized: yes
     :setup: Standalone Instance
     :steps:
         1. Add a new attribute type with a superior
@@ -244,6 +248,7 @@ def test_edit_attribute_keep_custom_values(create_attribute):
     """Test editing a custom schema attribute keeps all custom values
 
     :id: 5b1e2e8b-28c2-4f77-9c03-07eff20f763d
+    :parametrized: yes
     :setup: Standalone Instance
     :steps:
         1. Create a custom attribute
diff --git a/dirsrvtests/tests/suites/mapping_tree/regression_test.py b/dirsrvtests/tests/suites/mapping_tree/regression_test.py
index 51c687059..2c57c2973 100644
--- a/dirsrvtests/tests/suites/mapping_tree/regression_test.py
+++ b/dirsrvtests/tests/suites/mapping_tree/regression_test.py
@@ -111,6 +111,7 @@ def test_sub_suffixes(topo, orphan_param):
     """ check the entries found on suffix/sub-suffix
 
     :id: 5b4421c2-d851-11ec-a760-482ae39447e5
+    :parametrized: yes
     :feature: mapping-tree
     :setup: Standalone instance with 3 additional backends:
             dc=parent, dc=child1,dc=parent, dc=childr21,dc=parent
diff --git a/dirsrvtests/tests/suites/password/password_test.py b/dirsrvtests/tests/suites/password/password_test.py
index 2d4aef028..94a23e669 100644
--- a/dirsrvtests/tests/suites/password/password_test.py
+++ b/dirsrvtests/tests/suites/password/password_test.py
@@ -156,6 +156,7 @@ def test_pwd_scheme_no_upgrade_on_bind(topology_st, crypt_scheme, request, no_up
     the current hash is in nsslapd-scheme-list-no-upgrade-hash
 
     :id: b4d2c525-a239-4ca6-a168-5126da7abedd
+    :parametrized: yes
     :setup: Standalone instance
     :steps:
         1. Create a user with userpassword stored as CRYPT
diff --git a/dirsrvtests/tests/suites/replication/regression_m2_test.py b/dirsrvtests/tests/suites/replication/regression_m2_test.py
index 10a5fa419..68966ac49 100644
--- a/dirsrvtests/tests/suites/replication/regression_m2_test.py
+++ b/dirsrvtests/tests/suites/replication/regression_m2_test.py
@@ -991,6 +991,7 @@ def test_change_repl_passwd(topo_m2, request, bind_cn):
        Testing when agmt bind group are used.
 
     :id: a305913a-cc76-11ec-b324-482ae39447e5
+    :parametrized: yes
     :setup: 2 Supplier Instances
     :steps:
         1. Insure agmt from supplier1 to supplier2 is properly set to use bind group
diff --git a/dirsrvtests/tests/suites/vlv/regression_test.py b/dirsrvtests/tests/suites/vlv/regression_test.py
index d9d1cb444..f7847ac74 100644
--- a/dirsrvtests/tests/suites/vlv/regression_test.py
+++ b/dirsrvtests/tests/suites/vlv/regression_test.py
@@ -775,6 +775,7 @@ def test_vlv_reindex(topology_st, prefix, basedn):
     """Test VLV reindexing.
 
     :id: d5dc0d8e-cbe6-11ee-95b1-482ae39447e5
+    :parametrized: yes
     :setup: Standalone instance.
     :steps:
         1. Cleanup leftover from previous tests
@@ -830,6 +831,7 @@ def test_vlv_offline_import(topology_st, prefix, basedn):
     """Test VLV after off line import.
 
     :id: 8732d7a8-e851-11ee-9d63-482ae39447e5
+    :parametrized: yes
     :setup: Standalone instance.
     :steps:
         1. Cleanup leftover from previous tests
-- 
2.49.0

